migration-engine/connectors/sql-migration-connector/src/sql_renderer.rs

```diff
@@ -36,6 +37,65 @@ pub(crate) trait SqlRenderer {
     /// Render a `CreateEnum` step.
     fn render_create_enum(&self, create_enum: &CreateEnum) -> Vec<String>;
 
+    /// Render a `CreateTable` step.
+    fn render_create_table(
+        &self,
+        table: &Table,
+        schema_name: &str,
+        next_schema: &SqlSchema,
+        sql_family: SqlFamily,
+    ) -> anyhow::Result<Vec<String>> {
+        let columns: String = table
```

Comment:
This is basically copy-pasted for now. I think we can make this a lot nicer in the future by:

- Passing a TableRef here
- Specializing it per database. SQLite has very specific requirements. We can still share a lot of code as functions in the `common` module.